app/helpers/application_helper.rb

```diff
@@ -1,15 +1,20 @@
 module ApplicationHelper
-    def error_msg(resource, field)
-        if resource.errors.any?
-          return resource.errors.full_messages_for(field).first
-        end
-        ""
-      end
+  def error_msg(resource, field)
+    if resource.errors.any?
+      return resource.errors.full_messages_for(field).first
+    end
+    ""
+  end
+
+  def style_error_class(resource, field, css)
+    if resource.errors[field].any?  
+      return (css+" border-danger").to_s
+    end
+    return css.to_s
+  end
 
-      def style_error_class(resource, field, css)
-        if resource.errors[field].any?  
-          return (css+" border-danger").to_s
-        end
-        return css.to_s
-      end
+  def display_profile resource, class_name, default_image
```

Comment:
can you change the name to be more relevant?
would it make more sense if it was renamed 'display_profile_pic' ? @Aboorva1

Reply:
Renamed it @iMagesh